core/storage/storage_models_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Oppia storage models."""
+import inspect
+
+from core.platform import models
+from core.tests import test_utils
+
+
+class StorageModelsTest(test_utils.GenericTestBase):
+    """Tests for Oppia storage models."""
+
+    def test_all_model_names_unique(self):
+        all_model_names = []
+        for name in models.NAMES.__dict__:
+            if '__' not in name:
+                all_model_names.append(name)
+
+        all_models = []
```

Comment:
names_of_ndb_model_subclasses

(these are names, not models)

Reply:
Done